### PR TITLE
Callable to closure refactor

### PR DESCRIPTION
**Pull request by mvorisek**
You must always pass a proper Closure instead of callable

### CODE REVIEW

File: src/Component/InlineEdit.php [hunk 1]
```diff
@@ -134,19 +134,15 @@ public function setModel(\atk4\data\Model $model)
...
-            if ($this->cb->triggered()) {
-                $value = $_POST['value'] ? $_POST['value'] : null;
-                $this->cb->set(function () use ($fx, $value) {
-                    return call_user_func($fx, $value);
-                });
-            }
+        if (!$this->autoSave) {
+            $value = $_POST['value'] ?? null;
+            $this->cb->set(function () use ($fx, $value) {
+                return call_user_func($fx, $value);
```
Comment (georgehristov) [inline src/Component/InlineEdit.php]: `return $fx($value);`?

File: src/Form/Control/DropdownCascade.php [hunk 2]
```diff
@@ -65,7 +65,7 @@ public function getNewValues(string $id): array
         $model = $this->cascadeControl->model->load($id)->ref($this->reference);
         $values = [];
         foreach ($model as $k => $row) {
-            if ($this->renderRowFunction && is_callable($this->renderRowFunction)) {
+            if ($this->renderRowFunction) {
                 $res = call_user_func($this->renderRowFunction, $row, $k);
```
Comment (georgehristov) [inline src/Form/Control/DropdownCascade.php]: `$res = $this->renderRowFunction($row, $k);`?

File: src/Form/Control/Lookup.php [hunk 3]
```diff
@@ -202,7 +202,7 @@ public function getData($limit = true): array
      */
     public function renderRow(\atk4\data\Model $row): array
     {
-        $renderRowFunction = is_callable($this->renderRowFunction) ? $this->renderRowFunction : [__CLASS__, 'defaultRenderRow'];
+        $renderRowFunction = $this->renderRowFunction ?? \Closure::fromCallable([static::class, 'defaultRenderRow']);
 
         return call_user_func($renderRowFunction, $this, $row);
```
Comment (georgehristov) [inline src/Form/Control/Lookup.php]: `return $renderRowFunction($this, $row);`?

**Review by georgehristov — APPROVED** [review]
👍

File: src/Component/InlineEdit.php (hunk 1 above)
Comment (mvorisek) [reply src/Component/InlineEdit.php]: I would normally write it like that, but on the other side `call_user_func*` is easier to search. But yes, lets never use call_user_func :)

File: src/Form/Control/Lookup.php (hunk 3 above)
Comment (mvorisek) [reply src/Form/Control/Lookup.php]: I prefer to pass args only once to be consistent

File: src/Form/Control/DropdownCascade.php (hunk 2 above)
Comment (mvorisek) [reply src/Form/Control/DropdownCascade.php]: https://github.com/atk4/ui/pull/1384#discussion_r462926547

File: src/Component/InlineEdit.php (hunk 1 above)
Comment (mvorisek) [reply src/Component/InlineEdit.php]: done, yes, it is better

**Pull request merged**